App/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,109 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+    // switch to true to log data before validating 
+    private static final boolean DEBUGGING = true;
```

Comment:
Good idea to have this! We actually do something similar at work, though we have it as a command-line argument, and the behavior is slightly different.

Reply:
That would be awesome if I could figure out a way to set that variable to true when running the mvn run command (if it's possible). That little "trick" has saved me some hours debugging and it is awesome to hear you actually use at least something similar in a professional setting!

Reply:
Oh it should totally be doable! I have a couple ideas we can flesh out later if you're interested, but to start with I'd suggest googling environment variables and how to use them. ;)